.github/workflows/graph_db_tests.yml

```
@@ -66,6 +66,67 @@ jobs:
+          --health-timeout 5s
+          --health-retries 5
+    steps:
+      - name: Check out
+        uses: actions/checkout@v6
+        with:
+          fetch-depth: 0
+
+      - name: Cognee Setup
+        uses: ./.github/actions/cognee_setup
+        with:
+          python-version: ${{ inputs.python-version }}
```

actionlint (1.7.11)

[error] property "python-version" is not defined in object type {ci-image: string; databases: string}

(expression)